File: SecondYear/itsrishti_SrishtiSingh_2226CSE(AI&ML)1107_2/Week_2/count_digits.java
public class count_digits{
    /**
     * @param n
     * @return
     */
    public static int countdigits(int n){
      int temp = n;
      int digit =0;
      int count =0;
      while(temp!=0){
          digit = temp%10;
          if(digit !=0 && n%digit ==0){
              count++;
          }
          temp=temp/10;
      }
      return count;
          }
      }

File: SecondYear/chaudharyaryanpanwar_AryanPanwar_2226cs1053_2/Week_2/CountDigits.cpp
int countDigits(int n){
	// Write your code here.
	int count = 0 ; 	
	int temp  = n ;
	int len = 0 ;
	while (temp != 0){
		temp /= 10;
		len++;
	}
	temp = n ; 
	int last_digit;
	int factor;
	for (int i = 0 ; i< len ; i++){
		last_digit = temp%10; 
		if (last_digit == 0)count = count;
		else if (n%last_digit==0)
		{
			count++;
		}
		temp = temp/10;
	}
	return count;
}